index.js

```diff
@@ -34,21 +26,24 @@ function get(id) {
   if (!id) throw new Error('No fixture id provided');
 
   // add prefix zeros if they don't exist
-  id = (typeof id === 'number') ? getID(id) : id;
+  id = (typeof id === 'number') ? util.getID(id) : id;
 
   let final = {};
   let fixture;
+
   try {
-    fixture = require(`./src/${id}.js`)(mvt);
+    fixture = require(`./src/${id}.js`);
   } catch(err) {
-    throw new Error(`${id} is not a fixture`);
+
+    throw new Error(`Error loading fixture /src/${id}.js: ${err}`);
   }
 
   final.id = id;
   final.description = fixture.description;
   final.specification_reference = fixture.specification_reference;
   final.json = fixture.json;
-  final.buffer = generateBuffer(fixture.json);
+  final.proto = fixture.proto;
+  final.buffer = generateBuffer(fixture.json, fixture.proto);
```

Comment:
I noticed you removed `manipulate` from the fixtures, but left it here. I like that: we may still want to hook into it for certain fixtures where adding bad data is not easy via just an alternative scheme.

Reply:
heh, this was a mistake, but yeah might as well keep it! I'll make sure the starter template includes it with a note.